opentelemetry-otlp/src/retry.rs

```diff
@@ -161,13 +145,13 @@ where
                     RetryErrorType::Throttled(server_delay) if attempt < policy.max_retries => {
                         attempt += 1;
                         // Use server-specified delay (overrides exponential backoff)
-                        otel_warn!(name: "OtlpRetry", message = format!("Retrying operation {:?} after server-specified throttling delay: {:?}", operation_name, server_delay));
+                        otel_info!(name: "Export.InProgress.Throttled", operation = operation_name, error = format!("{:?}", err), attempt = attempt, delay_ms = server_delay.as_millis());
                         runtime.delay(server_delay).await;
                         // Don't update exponential backoff delay for next attempt since server provided specific timing
                     }
                     _ => {
                         // Max retries reached
-                        otel_warn!(name: "OtlpRetry", message = format!("Operation {:?} failed after {} attempts: {:?}", operation_name, attempt, err));
+                        otel_warn!(name: "Export.Failed.Exhausted", operation = operation_name, error = format!("{:?}", err), retries = attempt);
```

Comment:
otel_warn!(
                        name: "Export.Retry.Exhausted",
                        operation = operation_name,
                        retries = attempt,
                        message = "OTLP export failed after all retry attempts - telemetry data will be lost"
                    );